test/performance/perf_test.py

```diff
@@ -399,7 +399,7 @@ def insert_json_file_with_file_id(filename):
 
 def read_json_file(filename):
     coll = proc_client.perftest.corpus
-    temp = tempfile.TemporaryFile()
+    temp = tempfile.TemporaryFile(mode='w')
```

Comment:
This is required to fix the python 3 failure because TemporaryFile defaults to binary mode (https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryFile) but we are writing non-binary strings.